torch/csrc/distributed/rpc/agent_utils.cpp

```diff
@@ -111,11 +120,9 @@ std::unordered_map<std::string, worker_id_t> collectCurrentNames(
 
       nameToId.emplace(workerName, workerId);
     }
-    allWorkerInfos = fmt::format("{},{}-{}", allWorkerInfos, selfName, selfId);
-  } else {
-    // Add own name to worker list
-    allWorkerInfos = fmt::format("{}-{}", selfName, selfId);
   }
+  // Add own name to worker list
+  allWorkerInfos = fmt::format("{},{}-{}", allWorkerInfos, selfName, selfId);
```

Comment:
does it make sense to add the comma to the end instead of the beginning? i.e. `fmt::format("{}{}-{},", allWorkerInfos, selfName, selfId);` or `fmt::format("{}-{},{},", selfName, selfId, allWorkerInfos);`. In this way, it seems that the parsing can be easier, i.e., we no longer need to handle the last token separately?

Reply:
This makes more sense, I updated it to `fmt::format("{}{}-{},", allWorkerInfos, selfName, selfId);` and just ignore the last comma for the string when parsing.